revolv/payments/models.py

```diff
@@ -253,12 +261,55 @@ class RepaymentFragment(models.Model):
     factories = ImportProxy("revolv.payments.factories", "RepaymentFragmentFactories")
 
 
+class AdminAdjustmentManager(models.Manager):
+    """Simple manager for the AdminAdjustment model"""
+
+    def adjustments(self, start_date=datetime.datetime.min, end_date=datetime.datetime.max, cash_type=None):
+        queryset = AdminAdjustment.objects.all().order_by('created_at')
+        
+        if start_date and end_date:
+            queryset = queryset.filter(created_at__gte=start_date)
+            queryset = queryset.filter(created_at__lte=end_date)
+        if cash_type:
+            queryset = queryset.filter(cash_type=cash_type)
+        
+        return queryset
+
+class AdminAdjustment(models.Model):
+    """
+    Abstraction indicating one RE-volv adjustment to the accounting.
+    
+    This model exists for accounting reasons.
+    """
+    amount = models.FloatField("How many dollars is this transaction?")
+    name = models.CharField(
+            "What is the name of this expense or income?", 
```

Comment:
Oh no they shouldn't nevermind. I think the closing paren should be on a new line and de-indented though